### ISSUE TITLE
Zoom levels too low

### ISSUE THREAD

**Issue by plepe** [plepe/openstreetbrowser-categories-main#47]
Thanks!
BTW I noticed the minimum zoom level to be way too low to remain useful if you apply (such) filters. That's probably not only applicable to bicycle amenities but pretty much everything where you have set minimum zoom levels.

_Originally posted by @stefanct in https://github.com/plepe/openstreetbrowser-categories-main/issues/46#issuecomment-1201675118_

**Comment by plepe** [comment]
@stefanct Yes, you are right. Especially on a smartphone with small display. When setting the zoom levels, I had a desktop browser in mind.

There are two alternatives to solve this problem:
* Set higher zoom levels in the categories
* Add a setting, where you can override the current zoom level setting (force z12, although z17 is active)

Maybe both?

**Comment by stefanct** [comment]
I noticed on a laptop (w/o very high DPI)... I don't exactly see the connection between the device and the zoom level TBH, so maybe there was some miscommunication. What I meant is that the queries don't get executed when you zoom out too much. If there are many POIs this makes sense but as soon as you have only a few per city and want to get an overview of larger regions you are stuck and have to either pan endlessly or use overpass turbo etc. instead of OSB.

Why not just query and only show the results if they are below a certain limit (per pixel area)?

**Comment by plepe** [comment]
Sorry, forgot to reply.

The idea is to reduce traffic to the database server. So, I don't know about the density before querying the area ... and still, as I'm only asking for the first 250 (or so) result, I don't know if it is just over 250 or 10.000.

I'll think about improvements.
